test/table_test.rb

```diff
@@ -15,4 +15,39 @@
       table.candidate_parent_classes.must_equal []
     end
   end
+
+  describe "relation for association" do
+    let(:table) { PartialKs::Table.new(BlogPost) }
+
+    it "emits the correct foreign key for the table for belongs_to" do
+      table.relation_for_association(User).must_equal BlogPost.where(:user_id => 0)
+    end
+
+    it "use results from pluck for belongs_to" do
+      user_ids = [1,2,3]
+      User.stub :pluck, user_ids do
+        table.relation_for_association(User).must_equal BlogPost.where(:user_id => [0] + user_ids)
```

Comment:
Hmm, looks like I use hash rockets everywhere prior to this change, but where I use `:` it's used for passing in keyword arguments. I would suggest fixing any consistency if any, later ?